### DISCUSSION TITLE
Someone is checking the PR's?

### DISCUSSION THREAD

**Discussion by SoyDiego** [shadcn-ui/ui#1305] [General]
Hi people, I'm new here.
I did my first 2 PR's yesterday but I saw a lot of PR's with long time and nobody is approving or reviewing if something is good or not or needs to improve something.
It's not a criticism, I'm just writing to find out if there's any other way I can help and contribute :)

Thanks

**Comment by farishan** [comment]
Same thought here. Looking at the pull request count, I just feel like @shadcn should collaborate more with reviewers or someone like community managers. I believe many would volunteer :D
